web/modules/custom/dbcdk_community/src/Plugin/Block/FlaggedContentList.php

```diff
@@ -9,9 +9,7 @@
 
 use DBCDK\CommunityServices\ApiException;
 use Drupal\Core\Block\BlockBase;
-use Drupal\Core\Logger\LoggerChannelFactory;
 use Drupal\Core\Plugin\ContainerFactoryPluginInterface;
-use Drupal\dbcdk_community\Content\FlaggableContent;
```

Comment:
Hmm, I think we have to go with the fully qualified names then (not too keen about it though). But the trade off is a PHP CodeSniffer that will complain about unused classes